### ISSUE TITLE
Cannot create incident or metric

### ISSUE THREAD

**Issue by unixengineer** [CastawayLabs/cachet-monitor#14]
I am getting errors when I am trying to do a URL check .. 
Here is my config file

```
{
  "api_url": "http://10.82.91.107/api/v1",
  "api_token": "7lipnzcreXL2eXQTYS9a",
  "monitors": [
    {
      "name": "Some Jenkins Master",
      "url": "http://myserver.mydomain.com/jenkinss/",
      "metric_id": 1,
      "component_id": 4,
      "threshold": 80,
      "expected_status_code": 200
    }
  ]
}
```

```
2015/06/24 21:43:37 /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/monitor.go:77: Creating incident...
2015/06/24 21:43:37 /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/incident.go:42: Cannot parse incidents: json: cannot unmarshal string into Go value of type int
2015/06/24 21:43:37 /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/incident.go:69: 400 {"errors":[{"status":400,"title":"Bad Request","detail":"The request cannot be fulfilled due to bad syntax."}]}
2015/06/24 21:43:37 /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/incident.go:82: Could not create/update incident!
2015/06/24 21:43:38 /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/incident.go:109: Cannot parse component body.
panic: json: cannot unmarshal string into Go value of type int

goroutine 52 [running]:
github.com/castawaylabs/cachet-monitor/cachet.(*Incident).fetchComponent(0xc2080a2960, 0x0, 0x0)
    /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/incident.go:110 +0x2de
github.com/castawaylabs/cachet-monitor/cachet.(*Incident).UpdateComponent(0xc2080a2960)
    /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/incident.go:125 +0x66
github.com/castawaylabs/cachet-monitor/cachet.(*Monitor).AnalyseData(0xc208010230)
    /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/monitor.go:97 +0x8f1
github.com/castawaylabs/cachet-monitor/cachet.(*Monitor).Run(0xc208010230)
    /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/cachet/monitor.go:34 +0x1e0
created by main.main
    /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/main.go:26 +0x671

goroutine 1 [chan receive]:
main.main()
    /usr/local/go/bin/src/github.com/castawaylabs/cachet-monitor/main.go:24 +0x601

goroutine 61 [runnable]:
net/http.(*persistConn).readLoop(0xc2080662c0)
    /usr/local/go/src/net/http/transport.go:928 +0x9ce
created by net/http.(*Transport).dialConn
    /usr/local/go/src/net/http/transport.go:660 +0xc9f

goroutine 62 [select]:
net/http.(*persistConn).writeLoop(0xc2080662c0)
    /usr/local/go/src/net/http/transport.go:945 +0x41d
created by net/http.(*Transport).dialConn
    /usr/local/go/src/net/http/transport.go:661 +0xcbc

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
    /usr/local/go/src/runtime/asm_amd64.s:2232 +0x1
```

**Comment by matejkramny** [comment]
Hmm could be that the cachet API has changed since this was written.

**Comment by matejkramny** [comment]
Fixed with #15
